txn.go

```diff
@@ -334,6 +334,39 @@ func (txn *Txn) First(table, index string, args ...interface{}) (interface{}, er
 	return value, nil
 }
 
+// LongestPrefix is used to fetch the longest prefix match for the given
+// constraints on the index. Note that this will not work with the memdb
+// StringFieldIndex because it adds null terminators which prevent the
```

Comment:
Would it be possible to make a call to `PrefixFromArgs()` to strip the `\x00`? Since this only supports prefix indexers, seems like that should be possible.